Dockerfile

```diff
@@ -1,22 +1,43 @@
-FROM kbase/sdkbase2 as build
+FROM eclipse-temurin:11-jdk as build
+
+WORKDIR /tmp
+RUN apt-get update -y && \
+        apt-get install -y tzdata && apt-get install -y ca-certificates python3-sphinx ant git && \
+        git clone https://github.com/kbase/jars
 
 COPY . /tmp/workspace_deluxe
-RUN pip install configobj && \
-    cd /tmp && \
-    git clone https://github.com/kbase/jars && \
-    cd workspace_deluxe && \
+RUN cd workspace_deluxe && \
     make docker_deps
 
-FROM kbase/kb_jre
+# updated/slimmed down version of what's in kbase/kb_jre
+FROM ubuntu:18.04
 
 # These ARGs values are passed in via the docker build command
 ARG BUILD_DATE
 ARG VCS_REF
 ARG BRANCH=develop
 
+ENV DOCKERIZE_VERSION linux-amd64-v0.6.1
+ENV TOMCAT_VERSION tomcat8
+USER root
```

Comment:
Well this makes me nervous

Reply:
It's copied from the kbase/kb_jre Dockerfile. Presumably the root user is required to set up all the highly important directories and permissions that follow.

Reply:
Sorry, does the server run as root or not?

Reply:
yes, it does

Reply:
yuck, but not surprised